tests/v2/validation/networking/connectivity/network_policy.go

```diff
@@ -64,3 +66,15 @@ func newTestContainerMinimal() corev1.Container {
 	pullPolicy := corev1.PullAlways
 	return workloads.NewContainer(containerName, containerImage, pullPolicy, nil, nil, nil, nil, nil)
 }
+
+// curlCommand is a helper to run a curl command on an SSH shell node
```

Comment:
can you specify that this only works on rke2/k3s nodes?

Reply:
it actually works in all nodes types